kola/tests/misc/ntp.go

```diff
@@ -62,13 +60,12 @@ func NTP(c cluster.TestCluster) error {
 			return fmt.Errorf("unexpected systemd-timesyncd status: %v", out)
 		}
 
-		plog.Info("systemd-timesyncd.service is working!")
 		return nil
 	}
 
 	err = util.Retry(60, 1*time.Second, checker)
 	if err != nil {
-		return nil
+		return err
```

Comment:
I just went with a minor change because in a future PR I'm migrating from returning errors to calling Errorf/Fatalf and friends so this line of code is about to change again. I didn't want actual fixes to be hidden in that noise.